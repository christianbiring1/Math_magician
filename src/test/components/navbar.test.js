import React from 'react';
import renderer from 'react-test-renderer';
import Navbar from '../../components/navbar';

it('matches Navbar snapshot', () => {
  const tree = renderer.create(
    <Navbar />,
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
