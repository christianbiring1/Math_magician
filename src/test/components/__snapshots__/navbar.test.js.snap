// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches Navbar snapshot 1`] = `
<div
  className="navbar"
>
  <h1
    className="title"
  >
    Math Magicians
  </h1>
  <nav>
    <ul
      className="nav-list"
    >
      <li
        className="nav-items"
      >
        <a
          href="./Welcome"
        >
          Home
        </a>
      </li>
      <li
        className="nav-items item-line"
      >
        <a
          href="./Calculator"
        >
          Calculator
        </a>
      </li>
      <li
        className="nav-items item-line"
      >
        <a
          href="./Quote"
        >
          Quote
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
